client/src/App.css

```diff
@@ -1,30 +1,20 @@
 .App {
-  text-align: center;
   background-image: radial-gradient(circle farthest-side at top center, #434343 0%, #2F2C2C 36%);
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-.heading-font {
-  font-family: "Nunito Sans", sans-serif;
+footer {
+  text-align: center;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+* {
```

Comment:
To order this by importance, let's put `*` first, then `.App`, then `main`, sketchpicker and lightbox sub, then lastly `footer`

Reply:
ok done